linux_os/guide/services/ssh/ssh_server/sshd_use_approved_ciphers/oval/shared.xml

```diff
@@ -38,7 +49,11 @@
   </ind:textfilecontent54_test>
   <ind:textfilecontent54_object id="obj_sshd_use_approved_ciphers" version="2">
     <ind:filepath>/etc/ssh/sshd_config</ind:filepath>
+    {{% if product in ['opensuse', 'sle11', 'sle12'] %}}
+    <ind:pattern operation="pattern match">^[\s]*(?i)Ciphers(?-i)[\s]+((aes128-ctr|aes192-ctr|aes256-ctr),?)+[\s]*(?:|(?:#.*))?$</ind:pattern>
+    {{% else %}}
```

Comment:
NACK on this change. FIPS ciphers are mandated by Federal law.

Reply:
SLE may have different FIPS validated ciphers. Their OpenSSH paperwork should confirm which ciphers were evaluated.

Reply:
Their FIPS validated ciphers are pretty much the same.